recipes/arro3/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+arro3_module:
+  - arro3-core
+  # - arro3-compute
+  # - arro3-io
```

Comment:
Question for maintainers @conda-forge/help-python: arro3 is a namespace package, so there will be multiple internal modules that are published separately, but expected to interoperate and to all use the same version. There may be more namespace modules in the future. Is it ok to have a single conda-forge recipe for all modules? Or do I need to have a different recipe for each namespace module.